.circleci/config.yml

```diff
@@ -62,23 +56,66 @@ jobs:
           name: mgmt - Validate Terraform
           command: cd terraform/mgmt && terraform validate
 
-      - run:
-          name: env - Create Terraform variables file
-          command: cd terraform/env && cp terraform.tfvars.example terraform.tfvars
       - run:
           name: env - Set up Terraform
-          command: cd terraform/env && terraform init -backend=false
+          command: cd terraform/env && terraform init "-backend-config=bucket=$TF_ENV_BUCKET"
       - run:
           name: env - Validate Terraform
           command: cd terraform/env && terraform validate
+      - persist_to_workspace:
+          root: .
+          paths:
+            - terraform/env/.terraform/*
+
+  deploy_env:
+    docker:
+      - image: 18fgsa/devsecops-builder:alpine
```

Comment:
On the fence about whether to use this image across all the jobs. Pro is that we can then ensure that the Terraform/Packer/Ansible versions are the same, con is that the other jobs might be a bit slower because they have to pull a larger image. Not a huge difference, I suppose.